### PR TITLE
[story/ECP-854] - V7-V10 Update: Content change to 'Apply for an early-career payment'

### PR DESCRIPTION
**Pull request by martikat**

### CODE REVIEW

File: app/views/static_pages/landing_page.html.erb [hunk 1]
```diff
@@ -23,59 +23,59 @@
     </p>
     <p class="govuk-body">
       If you want to know more about the qualifying criteria, check our 
-      <a class="govuk-link" href="/">eligibility page.</a>
+      <a class="govuk-link" href="/">eligibility page</a>.
```
Comment (WorkSutton) [inline app/views/static_pages/landing_page.html.erb:26]: I think the eligibility page needs updating? Pretty certain its in master now so possibly just a rebase

**Review by WorkSutton — CHANGES_REQUESTED** [review]

File: app/views/static_pages/landing_page.html.erb [hunk 2]
```diff
@@ -23,59 +23,59 @@
...
-      <%= render partial: "shared/svg_for_govuk_button_start_icon" %>
-    <% end %>
-
-    <h2 class="govuk-heading-l">Future claims</h2>
+    <h2 class="govuk-heading-l">Who can apply in the future?</h2>
 
     <p class="govuk-body">
       If you are one of the below teachers, you can still 
-      <a class="govuk-link" href="/">check your eligibity today.</a>
+      <a class="govuk-link" href="/">check your eligibility today</a>.
```
Comment (WorkSutton) [inline app/views/static_pages/landing_page.html.erb:33]: Same as above with the link

File: app/views/static_pages/landing_page.html.erb (hunk 1 above)
Comment (martikat) [reply app/views/static_pages/landing_page.html.erb]: I've updated the link

File: app/views/static_pages/landing_page.html.erb (hunk 2 above)
Comment (martikat) [reply app/views/static_pages/landing_page.html.erb]: I've updated the link

**Review by WorkSutton — APPROVED** [review]

**Pull request merged**